pandas/core/indexes/multi.py

```diff
@@ -30,6 +30,7 @@
 from pandas.core.dtypes.generic import ABCDataFrame
 from pandas.core.dtypes.missing import array_equivalent, isna
 
+from pandas._typing import OneOrMoreObjects
```

Comment:
I think OK to leave this out of `pandas._typing` for now - just a thought for the future